bylaws/programs/04-operations.tex

```diff
@@ -0,0 +1,63 @@
+% SECTION G: OPERATIONS PROGRAM
+\Section{Operations Program}
+
+\Subsection{Definitions}
+
+\begin{enumerate}
+    \item The use of the term "RITSEC Cloud" in this context collectively
+        refers to any RITSEC-owned compute, storage, and networking resources.
+    \item The use of the term "the Program" in this context refers to the
+        Operations Program.
+\end{enumerate}
+
+\Subsection{Purpose}
+
+\begin{enumerate}
+    \item The intent of the Program is to make it possible to maintain a highly
+        available cloud for RITSEC club operations and the RITSEC members.
+    \item The organization and operation of the Program is intended to reduce
+        dependencies on individual people for the successful operation of the
+        RITSEC cloud.
+\end{enumerate}
+
+\Subsection{Membership Requirements}
+
+\begin{enumerate}
+    \item The individual who holds the role of RITSEC Operations Lead will also
+        hold the position of Operations Chief.
```

Comment:
We may wish to consider moving this to the definition of the role, as for Interest Group Chief is linked in the definition of Secretary. If we decided this should stay here then the IG Chief should be moved (one needs to occurr for consistency). Both have merits; defined at the EBoard role gives one spot for people to see their duties but would fragment where information about a new IP would be need to put.